documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_writing.adoc

```diff
@@ -6,7 +6,7 @@ We are keen to accept and distribute interesting post-processing stages contribu
 
 ==== Basic Post-Processing Stages
 
-Post-processing stages have a simple API, and users can create their own by deriving from the `PostProcessingStage` class. The member functions that must be implemented are listed below, though note that some be be unnecessary for simple stages.
+Post-processing stages have a simple API, and users can create their own by deriving from the `PostProcessingStage` class. The member functions that must be implemented are listed below, though note that may be unnecessary for simple stages.
```

Comment:
Try again. I believe it was supposed to be "though note that some may be unnecessary for simple stages."

Reply:
@lurch 
Ah...I misunderstood. fixed it. :laughing: